sci-geosciences/qgis/qgis-9999.ebuild

```diff
@@ -136,6 +137,7 @@ src_configure() {
 		-DWITH_GEOREFERENCER=$(usex georeferencer)
 		-DWITH_GRASS7=$(usex grass)
 		$(cmake-utils_use_find_package hdf5 HDF5)
+		-DWITH_GUI=ON
```

Comment:
these lines are not sorted by upstream cmake variable names, but unconditional vars, then conditional USE flag name. please move it up to the other unconditional vars then.

Reply:
Moved up.